src/views/clusteroverview/SettingsTab/ClusterTab/ClusterTab.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { FC, useEffect, useState } from 'react';
+
+import Loading from '@kubevirt-utils/components/Loading/Loading';
+import { useIsAdmin } from '@kubevirt-utils/hooks/useIsAdmin';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { usePreviewFeatures } from '@kubevirt-utils/hooks/usePreviewFeatures/usePreviewFeatures';
+import { Checkbox, Divider } from '@patternfly/react-core';
+
+import GeneralTab from '../GeneralTab/GeneralTab';
+import LiveMigrationTab from '../LiveMigrationTab/LiveMigrationTab';
+import TemplatesProjectTab from '../TemplatesProjectTab/TemplatesProjectTab';
+
+import ExpandSection from './ExpandSection';
+
+import './ClusterTab.scss';
+
+const ClusterTab: FC = () => {
+  const { t } = useKubevirtTranslation();
+
+  const { instanceTypesEnabled, loading, toggleInstanceTypesFeature } = usePreviewFeatures();
```

Comment:
Hmm it don't feel comfortable to me that this hook need to be combined with admin hook all the time. Maybe add useadmin to this hook and return canEdit or something like that?
Another thing is that this hook is very specific to instance type which in my opinion is a miss. As tomorrow can be another feature with preview tech that we may want to enable or disable. Think about making this hook more generic by getting a key and returning based on that key. So in this configmap not only one preview tech can live.

Reply:
This can be done in a follow up or for sure

Reply:
you're right, modifed to accept a featureName as key to have a more generic approach, also added isAdmin hook inside, thank you very much! :)